src/lucky_flow/drivers/chrome.cr

```diff
@@ -12,26 +21,19 @@ class LuckyFlow::Drivers::Chrome < LuckyFlow::Driver
     @driver.try(&.stop)
   end
 
-  protected def args : Array(String)
-    [] of String
-  end
-
-  @driver : Selenium::Driver?
-
-  private def driver : Selenium::Driver
-    @driver ||= begin
-      service = Selenium::Service.chrome(driver_path: driver_path)
-      Selenium::Driver.for(:chrome, service: service)
-    end
-  end
-
   private def driver_path
     LuckyFlow.settings.driver_path || Webdrivers::Chromedriver.install
   rescue err
     raise DriverInstallationError.new(err)
   end
+end
+
+LuckyFlow::Registry.register :chrome do
+  LuckyFlow::Drivers::Chrome.new { }
+end
 
-  private def browser_binary : String?
-    LuckyFlow.settings.browser_binary
+LuckyFlow::Registry.register :headless_chrome do
```

Comment:
Don't like this config? Just call `LuckyFlow::Registry.register :headless_chrome do ... end` with your own setup and it will overwrite this one 👍